src/webview_view/queryResultPanel.ts

```diff
@@ -98,12 +98,14 @@ export class QueryResultPanel implements WebviewViewProvider {
   ) {
     this._panel = panel;
     this.setupWebviewOptions(context);
-    this.renderWebviewView(context);
+    await this.renderWebviewView(context);
     this.setupWebviewHooks(context);
-    await this.transmitConfig();
     _token.onCancellationRequested(async () => {
       await this.transmitReset();
     });
+    // Moving it after cancelation request
+    // seems to give us just enough time to make sure the webview becomes live
+    this.transmitConfig();
```

Comment:
You are right! I think the move didnt help but removing the await did. I'll move it back to the previous position. 
As for the await, it was causing the query panel to perpetually wait after posting a message to update the config. 

To reproduce the error:
revert the fix
start debugger and do not select the query panel upon start.
Once the project loads, then navigate to query panel. 
Observation: It seems to be stuck in loading.
This await causes the webview to render the loading animation (not sure if its actually waiting, it might just be that the animation does not catch that this action has finished - if its the former, we might not want to await the post message as well)